pkg/crc/preflight/preflight_darwin.go

```diff
@@ -7,6 +7,11 @@ import (
 
 // StartPreflightChecks performs the preflight checks before starting the cluster
 func StartPreflightChecks() {
+	preflightCheckSucceedsOrFails(false,
+		checkOcBinaryCached,
+		"Checking if oc binary is cached",
+		false,
```

Comment:
Shouldn't this warn?

at the moment thjis is not an essential part of the deploymen.

Reply:
I am going to send a follow up PR which will make it essential, want to make sure to check the cluster operators are running and state is as expected.